src/main/kotlin/com/hedvig/underwriter/serviceIntegration/memberService/MemberServiceClient.kt

```diff
@@ -69,8 +69,14 @@ interface MemberServiceClient {
     @PostMapping("_/member/start/sign/norwegian/bankid/{memberId}")
     fun startNorwegianSing(
         @PathVariable("memberId") memberId: Long,
-        @RequestBody request: UnderwriterStartNorwegianBankIdSignSessionRequest
-    ): ResponseEntity<StartNorwegianBankIdSignResponse>
+        @RequestBody request: UnderwriterStartRedirectBankIdSignSessionRequest
+    ): ResponseEntity<StartRedirectBankIdSignResponse>
+
+    @PostMapping("_/member/start/sign/danish/bankid/{memberId}")
+    fun startDanishSing(
```

Comment:
is `sing` or `sign`?
```suggestion
    fun startDanishSign(
```